Library/Formula/pmd.rb

```diff
@@ -25,6 +25,7 @@ def caveats; <<-EOS.undent
   end
 
   test do
+    ENV["_JAVA_OPTIONS"] = "-Duser.home=#{ENV["HOME"]}”
```

Comment:
I wasn't sure, sorry. I think `a` is the best answer for now. Sorry to send you back and forth.